From 3a6878893a0a4762606862949d11ca2b60ce9aec Mon Sep 17 00:00:00 2001
From: Nam Dang <nam.dang.uw@rvc.renesas.com>
Date: Tue, 29 Jan 2019 10:12:53 +0700
Subject: [PATCH 717/909] soc: renesas: rcar-sysc: Remove power domains always
 on for M3N v1.0

commit c898e9b8d8436eda899958aaefdf02c4d1597ccb from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

In the past, we have enabled power domains always on
for M3N v1.0 due to hardware restriction.
But after confirming from hardware team
it is not needed. So, this patch removes it.

Signed-off-by: Nam Dang <nam.dang.uw@rvc.renesas.com>
Signed-off-by: Dien Pham <dien.pham.ry@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/soc/renesas/rcar-sysc.c | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/drivers/soc/renesas/rcar-sysc.c b/drivers/soc/renesas/rcar-sysc.c
index 64c11374e521..7fabcfe0a666 100644
--- a/drivers/soc/renesas/rcar-sysc.c
+++ b/drivers/soc/renesas/rcar-sysc.c
@@ -91,12 +91,6 @@ const struct soc_device_attribute rcar_sysc_quirks_match[] __initconst = {
 			| BIT(R8A7796_PD_A3IR) | BIT(R8A7796_PD_3DG_A)
 			| BIT(R8A7796_PD_3DG_B)),
 	},
-	{
-		.soc_id = "r8a77965", .revision = "ES1.0",
-		.data = (void *)(BIT(R8A77965_PD_A3VP) | BIT(R8A77965_PD_CR7)
-			| BIT(R8A77965_PD_A3VC) | BIT(R8A77965_PD_A2VC1)
-			| BIT(R8A77965_PD_A3IR)),
-	},
 	{ /* sentinel */ }
 };
 
-- 
2.17.1

